### ISSUE TITLE
Deletion of all DNSSEC keys via EPP

### ISSUE THREAD

**Issue by jonasbn** [DK-Hostmaster/Roadmap#14]
This is described in [RFC:5910](https://tools.ietf.org/html/rfc5910#page-18), which describes the extension for DNSSEC handling in  EPP

**Comment by jonasbn** [comment]
Example from [RFC:5910 page 23](https://tools.ietf.org/html/rfc5910#page-23)

```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<epp xmlns="urn:ietf:params:xml:ns:epp-1.0"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <command>
    <update>
      <domain:update xmlns:domain="urn:ietf:params:xml:ns:domain-1.0">
        <domain:name>example.com</domain:name>
      </domain:update>
    </update>
    <extension>
      <secDNS:update urgent="true"
        xmlns:secDNS="urn:ietf:params:xml:ns:secDNS-1.0">
        <secDNS:rem>
          <secDNS:all>true</secDNS:all>
        </secDNS:rem>
      </secDNS:update>
    </extension>
    <clTRID>ABC-12345</clTRID>
  </command>
</epp>
```

DK Hostmaster does not plan to support the `urgent` parameter at this time, meaning it will not be effectuated if defined and if set to `true` the error code: `2102` will be returned as specified in RFC:5910.

**Comment by jonasbn** [comment]
The feature is available for EPP with release 3.2.0., which is currently deployed to production and sandbox, please see the environment details page in the [EPP Service Wiki](https://github.com/DK-Hostmaster/epp-service-specification/wiki).
